cypress/e2e/invalidlogin.cy.js

```diff
@@ -0,0 +1,16 @@
+describe('Login with invalid credentials and get error message', () => {
+  it('login', () => {
+    cy.visit('https://emel-h.github.io/social-media-client/')
```

Comment:
```suggestion
    cy.visit('/')
```

It's very important to test the local code in the branch, not the remote code that is deployed. At the moment you are testing a static build, not the most up to date code.